server/templates/deepseek_fim.yaml

```diff
@@ -1,13 +1,15 @@
+# Since deepseek_fim_client.py makes requests with a prefix/suffix directly, `prompt` and `args` are not utilized.
+
 prompt: |
   {prefix}
 
 stop_tokens:
   - "\n\n"
   - "\n \n"
-  - "<|eos_token|>"
-  - "<|fim_begin|>"
-  - "<|fim_hole|>"
-  - "<|fim_end|>"
+  - "<｜end▁of▁sentence｜>"
```

Comment:
Where is the documentation for this token? The paper (https://arxiv.org/pdf/2401.14196) uses eos_token.

Reply:
From its tokenizer_config.json: https://huggingface.co/deepseek-ai/DeepSeek-V3-Base/blob/main/tokenizer_config.json